Automattic/wp-calypso
# Help: Track when customers contact us while they have active tickets

Part of pbvpgB-jw-p2. 

We want to see how often customers are contacting us when they already have open tickets. In followup PRs, we'll run some experiments that will attempt to reduce this number by letting customers know they have open tickets (anticipating a better experience for them to confirm they still have an open request, and reduce ticket/chat volume for us).

To do so, this PR:

- Sets up Redux/data-layer scaffolding to fetch and set support history
- Creates a Query component to get this data
- Embeds the Query component where the form gets seen (both Inline Help and the full Contact Us page)
- Uses the support history to fire Tracks events when support is requested while there are active tickets

### How to test
- With no support tickets open, try the various support types (Chat, Email, Forum, Directly) — you shouldn't see any Tracks events fire
- With a New support ticket open, try the various support types (Chat, Email, Forum, Directly) — you should see a Tracks event fire noting # of active tickets and the right support type
- Try this with tickets in all states — New/Open/Hold tickets should trigger, but Pending/Solved/Closed tickets shouldn't.
